config/backend-config/types.go

```diff
@@ -48,6 +48,7 @@ type DestinationT struct {
 	DestinationDefinition DestinationDefinitionT
 	Config                map[string]interface{}
 	Enabled               bool
+	WorkspaceID           string
```

Comment:
Is this a new field returned by the control plane?

Reply:
It was there always, we are not using it before. No change done in CP on this.